prep.sh

```diff
@@ -66,11 +66,16 @@ git submodule update --init --remote
 # reference the latest tagged release. Other projects (like the
 # ETL or aggregator) don't do normal releases and so can just
```

Comment:
i guess update this comment to just reference the aggregator?

Reply:
Good point, done